libcontainer/cgroups/ebpf/devicefilter/devicefilter_test.go

```diff
@@ -83,71 +83,69 @@ block-2:
         19: Exit
 block-3:
         20: JNEImm dst: r2 off: -1 imm: 2 <block-4>
-        21: JNEImm dst: r4 off: -1 imm: 5 <block-4>
-        22: JNEImm dst: r5 off: -1 imm: 1 <block-4>
+        21: JNEImm dst: r4 off: -1 imm: 1 <block-4>
```

Comment:
How are these changes introduced?

Reply:
The removal of the `/dev/console` rule means that `5:1` is no longer in the default whitelist, which changes the code generated.